bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/provider/i18n/ModuleTypeI18nServiceImpl.java

```diff
@@ -76,35 +86,35 @@ public ModuleType getModuleTypePerLocale(ModuleType defModuleType, Locale locale
         String ldescription = ModuleTypeI18nUtil.getLocalizedModuleTypeDescription(i18nProvider, bundle, uid,
                 defModuleType.getDescription(), locale);
 
-        List<ConfigDescriptionParameter> lconfigDescriptions = getLocalizedConfigDescriptionParameters(
+        List<ConfigDescriptionParameter> lconfigDescriptionParameters = getLocalizedConfigDescriptionParameters(
                 defModuleType.getConfigurationDescriptions(), ModuleTypeI18nUtil.MODULE_TYPE, uid, bundle, locale);
         if (defModuleType instanceof ActionType) {
-            return createLocalizedActionType((ActionType) defModuleType, bundle, uid, locale, lconfigDescriptions,
-                    llabel, ldescription);
+            return createLocalizedActionType((ActionType) defModuleType, bundle, uid, locale,
+                    lconfigDescriptionParameters, llabel, ldescription);
         }
         if (defModuleType instanceof ConditionType) {
-            return createLocalizedConditionType((ConditionType) defModuleType, bundle, uid, locale, lconfigDescriptions,
-                    llabel, ldescription);
+            return createLocalizedConditionType((ConditionType) defModuleType, bundle, uid, locale,
+                    lconfigDescriptionParameters, llabel, ldescription);
         }
         if (defModuleType instanceof TriggerType) {
-            return createLocalizedTriggerType((TriggerType) defModuleType, bundle, uid, locale, lconfigDescriptions,
-                    llabel, ldescription);
+            return createLocalizedTriggerType((TriggerType) defModuleType, bundle, uid, locale,
+                    lconfigDescriptionParameters, llabel, ldescription);
         }
         return null;
     }
 
-    private List<ConfigDescriptionParameter> getLocalizedConfigDescriptionParameters(
-            List<ConfigDescriptionParameter> parameters, String prefix, String uid, Bundle bundle, Locale locale) {
-        URI uri = null;
+    private @Nullable List<ConfigDescriptionParameter> getLocalizedConfigDescriptionParameters(
+            List<ConfigDescriptionParameter> parameters, String prefix, String uid, Bundle bundle,
+            @Nullable Locale locale) {
         try {
-            uri = new URI(prefix + ":" + uid + ".name");
+            return localizationService
+                    .getLocalizedConfigDescription(bundle,
+                            new ConfigDescription(new URI(prefix + ":" + uid + ".name"), parameters), locale)
+                    .getParameters();
         } catch (URISyntaxException e) {
             logger.error("Constructed invalid uri '{}:{}.name'", prefix, uid, e);
+            return null;
```

Comment:
We should return the default values here. I fixed it in https://github.com/openhab/openhab-core/pull/1161/files#diff-d90b601b2b68bbe8017ba644c98c6da1R387.